app/controllers/idv/hybrid_handoff_controller.rb

```diff
@@ -42,9 +47,11 @@ def self.selected_remote(idv_session:)
       if IdentityConfig.store.in_person_proofing_opt_in_enabled &&
          IdentityConfig.store.in_person_proofing_enabled &&
          idv_session.service_provider&.in_person_proofing_enabled
-        idv_session.skip_doc_auth == false
+        idv_session.skip_doc_auth == false ||
+          idv_session.skip_doc_auth_from_handoff == true
```

Comment:
Why are we using this value as a precondition when it's not set to true until the document capture page?

Reply:
@JackRyan1989 , it's for the "back" button to work from document capture(IPP) page. 

So the user can go from handoff (select ipp)-> document catpure -> Back -> handoff.

This is a new flag to control this behavior similar to `skip_doc_auth`.